packages/core/components/InputOrGroup/styles.module.css

```diff
@@ -48,7 +48,8 @@
   border-color: var(--puck-color-grey-8);
   border-radius: 4px;
   font-family: inherit;
-  padding: 12px 16px;
+  font-size: 16px;
```

Comment:
I find this size a bit uncomfortable on desktop. If this is required for mobile, can we restrict by viewport?

Reply:
Not really, as it's specifically an iOS touch thing, so should not be conflated with viewport.

Reply:
This is the only change I'm unsure on. The font-size feels way too chunky on desktop viewports to me now, in particular because the input font size is larger than the surrounding body font size.

<img width="285" alt="image" src="https://github.com/measuredco/puck/assets/985961/eb6e5556-974d-4c80-9f78-f4fce8f2d7e3">

I expect desktop to be the primary medium and would rather not compromise the consistency at this time.

How would you feel about either:

1. Leaving the font-size as-is and letting iOS zoom,
2. Only changing the font-size on smaller viewports, but accept that it will zoom on larger iOS devices (perhaps conflation, but I think that's an okay compromise) or
3. Prevent iOS from zooming with [`maximum-scale=1`](https://stackoverflow.com/a/46254706/1703260)

Reply:
I'll remove for now, cause not directly related to the PR, but suspect we'll need to return to this!

Reply:
Done

Reply:
Ty!